### ISSUE TITLE
Can not download file with git lfs

### ISSUE THREAD

**Issue by MarianaAlmeida** [markus93/NN_calibration#7]
Hi,

I can not download pickle files from the repo because the bandwidth does not allow.
I would be ok with only one file in the "logits/" folder to try your work out.

I tried doing "git lfs pull --include "logits/probs_resnet110_c10_logits.p" but it did not work either.

What should I do.

Thanks
Mariana

**Comment by markus93** [comment]
Hey Mariana,

Thanks for the question, there is also an alternative way of downloading the logits from Google Drive https://drive.google.com/drive/folders/1e1ai-bKb7LukKShqpn3S_gYXJGzhUYgm?usp=sharing.

**Comment by nickvdw** [comment]
In case anyone tries to clone the repository to download the model weights and gets the following error:

`This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.`,

you can follow these steps to still download the model weights:

1. Fork the repo to one of your users
2. Go to repo settings
3. Find "Include Git LFS objects in archives" under the Archives section and check it
4. Go to the Danger Zone section,  select "Archive this repository"
5. Confirm and authorize.
6. Return to the archived repository.
7. Download as .zip
8. Download will pause for a minute or so before it starts downloading lfs objects. Wait and it should continue.

From: https://github.com/nabla-c0d3/nassl/issues/17#issuecomment-804149931
